fe/fe-core/src/main/java/org/apache/doris/analysis/FunctionCallExpr.java

```diff
@@ -1148,12 +1148,12 @@ public void analyzeImpl(Analyzer analyzer) throws AnalysisException {
             for (int i = 0; i < children.size(); i++) {
                 if (children.get(i).type != Type.BOOLEAN) {
                     throw new AnalysisException("All params of "
-                        + fnName + " function must be boolean");
+                            + fnName + " function must be boolean");
```

Comment:
maybe just modify the part you need to change, otherwise all the developers need to rebase.

Reply:
Some of them were modified for the checkstyle failure, I will check in detail later.